### ISSUE TITLE
black screen playing video files

### ISSUE THREAD

**Issue by rkitover** [mpv-android/mpv-android#572]
### Important information

Android version:

mpv-android version:
0.34.0.-386-g284fecc0bd
latest armv7 build from github releases

### Description

On my Sony tv (tv info below), playing a file results in a black screen. Occasionally frames display when I pause/unpause. Also sometimes when I pause there is a lock icon in the top right corner. I tried with both hardware decoding and hardware decoding off.

Will provide logcat if you want, just need to set up wifi adb etc..

### Additional information

mpv.conf:

```console
profile=gpu-hq
scale=ewa_lanczossharp
cscale=ewa_lanczossharp
save-position-on-quit
```

![tv-info-2](https://lh3.googleusercontent.com/pw/AL9nZEXlfBu1IRNakd27r7S6GsuQsrtVhemajq8tKgsx4GcrBUqdcr6fckdha8KMg4RmitO-ts9mEcPlRpy0LIZ1PLCFI920574vdu9vtpJks_b0Lv72oFBeY86FXcPTSoPphI7PrgR5OT6hcJjVxDs-XhtA=w1920-h1440-no?authuser=0)
![tv-info-1](https://lh3.googleusercontent.com/pw/AL9nZEVSEbFPbGXvAgKDNWQLt74q5NL11yvAWh-no04h7_QIElZPCFbeoKPmn6hV3vJqkcvJ3unIX_lxNXvO1qc1etUvgQt-Bu5iGV7Qw7ENmAqRRS5GbPaoUyTfuURbszmY7y0tuIOrLTte6Sf6xJmngUET=w1920-h1440-no?authuser=0)

**Comment by sfan5** [comment]
`profile=gpu-hq` and especially the ewa scalers are demanding for hardware and it's not unusual for embedded GPUs to fail entirely, so far we haven't found any indication that this is mpv's fault.

**Comment by rkitover** [comment]
I tried completely uninstalling mpv-android and reinstalling the Google Play Store version and I have the same problem with no custom `mpv.conf`.

What is the lock icon in the upper right corner when I pause by the way?

**Comment by AwkwardOne** [comment]
You found a fix? I got the same problem on my Nokia streaming box.

**Comment by sfan5** [comment]
The fix is not enabling any advanced scaling or debanding. The default config should work on any device.
_Unless_ you have 10-bit video in which case `vf=format=yuv420p` might help. (#198)

**Comment by rkitover** [comment]
I tried `vf=format=yuv420p` but it did not help, I still have a black screen.

What is the lock icon in the upper right corner?

Here is a screenshot:

![screenshot of lock icon in mpv on tv](https://lh3.googleusercontent.com/pw/AL9nZEVJE7Ellf1SzfexPmbw16ECqwff_R0yzXwAvmMPGgJp9-nUn6QpWdQw5-lhAnIv2TGkT3J8Pi3qoXk5kJ_9OyBwL7DdPvn0h81YMx7N18WIaR2dJtOZMfEIadGEQbNr0L76XW6W6nBOyrkHcNc9wtIS=w1515-h1137-no?authuser=0)

**Comment by sfan5** [comment]
The lock icon allows entering a mode where touches are ignored until you unlock again, useless on a TV.

If you enable the stats view do you see them? If yes, please show a screenshot.
Which kind of file are you playing? Output from [mediainfo](https://mediaarea.net/MediaInfo) would be ideal.

**Comment by rkitover** [comment]
@sfan5 I figured out what I was doing wrong, when I was testing config changes it was still using the active mpv instance, so this time I rebooted the tv after every config change.

Both things you said turned out to be true in my case. I needed both `vf=format=yuv420p` in my config and I needed to remove `profile=gpu-hq/scale/cscale` as well. My final `mpv.conf` is:

```console
vf=format=yuv420p
save-position-on-quit
```
, now everything works!

It stutters a bit on start of the video but after that runs perfectly, and the picture is excellent.

I do wish I could use the high quality filter settings, I'll try that on my shield when I get it working again since it has a more powerful GPU.

Thank you for your work on this! I really appreciate having one of my favorite programs on Android. ❤️
